ui/app/adapters/transform.js

```diff
@@ -41,11 +41,11 @@ export default ApplicationAdapter.extend({
     return 'transform';
   },
 
-  urlForTransformations(backend, id) {
-    let url = `${this.buildURL()}/${encodePath(backend)}/transformation`;
-    if (id) {
-      url = url + '/' + encodePath(id);
-    }
+  urlForTransformations(backend, id, type) {
+    const base = `${this.buildURL()}/${encodePath(backend)}`;
+    // when type exists, transformations is plural
+    const url = type ? `${base}/transformations/${type}` : `${base}/transformation`;
+    if (id) return `${url}/${encodePath(id)}`;
```

Comment:
I find `url` easier to read this way, but I can revert to `url = url + '/' + encodePath(id);` if folks prefer that! I don't have strong feelings 😄